packages/react-components/react-accordion/src/components/AccordionItem/AccordionItem.tsx

```diff
@@ -5,6 +5,7 @@ import { renderAccordionItem_unstable } from './renderAccordionItem';
 import type { AccordionItemProps } from './AccordionItem.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 import { useAccordionItemStyles_unstable } from './useAccordionItemStyles';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { useAccordionItemStyles_unstable } from './useAccordionItemStyles';
import type { AccordionItemProps } from './AccordionItem.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```